src/handler.ts

```diff
@@ -1,3 +0,0 @@
-export async function handleRequest(request: Request): Promise<Response> {
-  return new Response(`request method: ${request.method}`)
-}
```

Comment:
Im thinking I should probably keep this pattern and keep the test that was there before... Not sure why I didn't like it 😅